library/src/containers/AsyncApi/AsyncApi.tsx

```diff
@@ -120,6 +123,7 @@ class AsyncApiComponent extends Component<AsyncApiProps, AsyncAPIState> {
         numberOfExpandedElement={initialExpandedElements}
       >
         <useChangeHashContext.Provider schemaName={bemClasses.getSchemaID()}>
+          <style>@import '/assets/async-api/fiori.css';</style>
```

Comment:
Could we make it more generic (is it possible)? If someone wants use not fiori, but custom css file, so what then? :)